hotline/users/views.py

```diff
@@ -207,3 +202,6 @@ def _edit(request, user=None):
     return render(request, 'users/edit.html', {
         "form": form,
     })
+
```

Comment:
`make test` will run flake8 for you, so you can avoid commit spurious characters.